Notification/notification.js

```diff
@@ -0,0 +1,136 @@
+        }
+
+        const blockHTML = `
+        <div class='notification-component'>
+        <form action="#" name='notificationForm' class='notification-component_form'>
+        <div class='notification-component_header'>
+            <button tabindex='1' name='notificationButtonClose' class='notification-button notification-button-close' accesskey='C'>X</button>
+        </div>       
+        <div class='notification-component_body'>
+            <p id='notificationTitle' class='notification-component_text notification-component_title'>
+            ${this.data[this.active].id}. ${this.data[this.active].title}                
+            </p>
+            <p id='notificationText' class='notification-component_text'>
+            ${this.data[this.active].phrase}   
+            </p>
+        </div>
+        <div class='notification-component_footer'>
+                <input tabindex='2' type="checkbox" name='notificationCheckBox' class="notification-form_check" accesskey='D'>
+                <label for="notificationCheckBox" class="notification-form-check_label">Disable</label>
+                <button tabindex='3' name='notificationButtonDecrease' class='notification-button notification-button-decrease' accesskey='W'><</button>
+                ${radioInputs}
+                <button tabindex='${this.data.length + 4}' name='notificationButtonIncrease' class='notification-button notification-button-increase' accesskey='S'>></button>
+            </form>
+        </div>
+    </div>`;
+
+        if (!('notificationDisable' in localStorage) || !JSON.parse(localStorage.getItem('notificationDisable'))) {
+            this.targetBlock.insertAdjacentHTML('afterbegin', blockHTML);
+        } else {
+            this.disable = true;
+        }
+    };
+
+    async initialize() {
+        await this.render()
+        if (!this.disable) {
+            const form = document.forms.notificationForm;
+            const closeButoon = form.elements.notificationButtonClose;
+            const decreaseButton = form.elements.notificationButtonDecrease;
+            const increaseButton = form.elements.notificationButtonIncrease;
+            const radioButton = form.elements.notificationRadio;
+            const checkBox = form.elements.notificationCheckBox;
+            const title = document.getElementById('notificationTitle');
+            const text = document.getElementById('notificationText');
+             
+            const setButtonState = () => {
+                if (this.active == 0) {
+                    decreaseButton.disabled = true;
+                } else {
+                    decreaseButton.disabled = false;
+                }
+                if (this.active == this.data.length - 1) {
+                    increaseButton.disabled = true;
+                } else {
+                    increaseButton.disabled = false;
+                }
```

Comment:
я бы сделал код чуть короче
```suggestion
                decreaseButton.disabled = this.active === 0;
                increaseButton.disabled = this.active === this.data.length - 1;
```